### ISSUE TITLE
melware

### ISSUE THREAD

**Issue by lIMonkieIl** [LarityRay/KeyAuth-Bypass-All-Programs#2]

**Comment by lIMonkieIl** [comment]
This is malware do not used it
![IMG_20220901_104713](https://user-images.githubusercontent.com/87721820/187877245-9cddc5f2-6a02-462c-b181-02aa5d1fd623.jpg)

**Comment by pp0236** [comment]
source is vaild and working only that hidden scr file is malware

**Comment by wnelson03** [comment]
@pp0236 no, the code was for the KeyAuth C++ example from a year ago. It won't work on the latest C++ example

We've already addressed this, it's the developer's responsibility to obfuscate and protect their program https://docs.keyauth.cc/security-practices
